insight_api/InsightAPI.py

```diff
@@ -0,0 +1,32 @@
+from contextlib import asynccontextmanager
+
+import uvicorn
+from fastapi import FastAPI, HTTPException
+from fastapi import status
+
+from insight_api.consts import MIN_RATINGS_FOR_INCLUSION, PORT, HOST
+from insight_api.log import logger
+from insight_api.postgres.AsyncPostgresConnection import AsyncPostgresConnection
+from insight_api.postgres.Review import find_top_rated_books
+
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Comment:
leave a simple comment explaining what is going on because the code here is not very explicit